hack/e2e.applications.sh

```
@@ -0,0 +1,47 @@
+for file in ./modules/*.sh; do
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)